storage/key_index.go

```diff
@@ -108,10 +108,10 @@ func (ki *keyIndex) get(atRev int64) (modified, created reversion, err error) {
 
 	n := g.walk(f)
 	if n != -1 {
-		return g.revs[n], g.created, nil
+		return g.revs[n], g.created, g.ver - int64(n), nil
```

Comment:
should it be `g.ver - (len(g.revs) - n)`? Because rev in `revs` is appended based on reversion. For example, generation{ver: 3, created: {50, 0}, revs: {{50, 0}, {60, 0}, {70, 0}}. the version of the third member is 3.